src/event_execution.jl

```diff
@@ -38,6 +39,24 @@ end
 #
 # transmissions
 #
+function execute!(::Val{OutsideTransmissionEvent}, state::SimState, params::SimParams, event::Event)::Bool
```

Comment:
I think we do not need an extra event type for this. `OutsideInfectionEvent` should be sufficient. It is better to have the extra logic in the function that is called at the beginning (something to supersede `initialfeed!` and simply schedule the future infections there

Reply:
ok, the current solution is not the worst, but converting it to a function will improve readability.